### PR TITLE
OPS: Use Forks

### PR DESCRIPTION
**Pull request by marcosrdz**
macOS Catalyst friendly forks

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/04epgt4z7m7et6a50jtz3r24gg

[download apk](http://filestorage.bluewallet.io:1488/1594453432.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pcdpud13kak9mq29uch98mzqzr

[download apk](http://filestorage.bluewallet.io:1488/1594526416.apk)

**Comment by marcosrdz** [comment]
Yup still needed. Gotta update it cause its been a while

**Comment by marcosrdz** [comment]
@Overtorment  updated.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/q386rb6a4qe7xh30cfqebt3far

[download apk](http://filestorage.bluewallet.io:1488/1597103292.apk)

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "@react-native-community/blur": "3.6.0",
     "@react-native-community/clipboard": "1.2.3",
     "@react-native-community/masked-view": "0.1.10",
-    "@react-native-community/push-notification-ios": "1.4.0",
+    "@react-native-community/push-notification-ios": "git+https://github.com/BlueWallet/push-notification-ios.git#e5e232117b02c7fc2170b682d2062fcbd2edd715",
```
Comment (Overtorment) [inline package.json:68]: whats wrong with the original?

Comment (marcosrdz) [reply package.json:68]: Contains  lines which are not compatible with Catalyst. I place an if macro between those lines

Comment (Overtorment) [reply package.json:68]: will you send PR to original repo with those changes?

Comment (marcosrdz) [reply package.json:68]: Most of the cases those packages are using deprecated unsupported methods. Example...
Image Picker uses assetslibrary. PHPhotoKit replaced that but the package isn't using it. The sections that make use of AssetsLibrary are in features we don't use so its safe to put if macros for catalyst. Same applies to RNCamera. Sending them a PR would result in PR Rejected given that I'm simply "commenting" features that others might use.

File: package.json [hunk 2]
```diff
@@ -111,7 +111,7 @@
     "react-native": "0.62.2",
     "react-native-biometrics": "git+https://github.com/BlueWallet/react-native-biometrics.git#f62b0b193e10376d4a0e8195c700a364ed4e5aaa",
     "react-native-blue-crypto": "git+https://github.com/Overtorment/react-native-blue-crypto.git",
-    "react-native-camera": "3.31.1",
+    "react-native-camera": "git+https://github.com/BlueWallet/react-native-camera.git#b733fff5d3d877ddbfe568af4f609ab6edfa5a09",
```
Comment (Overtorment) [inline package.json:114]: bad news is, Snyk is not gona send update PRs if its forked.
Im thinking, maybe its better to maintain parallel master branch for desktop only? `master-desktop`
and keep all that stuff there?

Comment (marcosrdz) [reply package.json:114]: Sounds good. Packages.json and podfile in that branch

**Pull request closed without merging**